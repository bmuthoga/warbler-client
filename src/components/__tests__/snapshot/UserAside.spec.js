import React from 'react'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import UserAside from '../../UserAside'

const props = {
  profileImageUrl: 'https://lorempixel.com/200/200/',
  username: 'testemail'
}

describe('UserAside Snapshot Tests', () => {
  it('should render the UserAside component', () => {
    const wrapper = shallow(
      <UserAside { ...props } />
    )

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
