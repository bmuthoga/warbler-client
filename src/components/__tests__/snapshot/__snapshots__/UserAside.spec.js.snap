// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserAside Snapshot Tests should render the UserAside component 1`] = `
<aside
  className="col-sm-2"
>
  <div
    className="panel panel-default"
  >
    <div
      className="panel-body"
    >
      <img
        alt="testemail"
        className="img-thumbnail"
        src="https://lorempixel.com/200/200/"
      />
    </div>
  </div>
</aside>
`;
